gateway/reverse_proxy.go

```diff
@@ -1478,13 +1518,21 @@ func copyBody(body io.ReadCloser) io.ReadCloser {
 }
 
 func copyRequest(r *http.Request) *http.Request {
+	if r.ContentLength == -1 {
```

Comment:
Add comments here, saying that this line for the case when request/response body is streaming.

Reply:
done